app/Http/Controllers/ConsultantController.php

```diff
@@ -40,4 +40,15 @@ public function getAllConsultant()
             "civil_engineers" => $civilEngineers];
         return response()->json($consultantsList, 200);
     }
+
+   public function getConsultantDetails($id){
+        $consultant = User::query()
+            ->join('consultants', 'users.id', '=', 'consultants.user_id')
+            ->where('users.id',$id)
+            ->get();
+
```

Comment:
please, handle the case of id is not found and add a status code for the response cases.